walkthroughs/howto-k8s-fargate/README.md

```diff
@@ -0,0 +1,53 @@
+## Prerequisites
+- Setup EKS cluster with Fargate support.
+  - You can use [clusterconfig.yaml](./clusterconfig.yaml) with [eksctl](https://eksctl.io)
+- [Walkthrough: App Mesh with EKS](../eks/)
+
+## Setup
+
+1. Clone this repository and navigate to the walkthrough/howto-k8s-alb folder, all commands will be run from this location
+2. **Your** account id:
+    ```
+    export AWS_ACCOUNT_ID=<your_account_id>
+    ```
+3. **Region** e.g. us-east-2
+    ```
+    export AWS_DEFAULT_REGION=us-east-2
+    ```
+4. **ENVOY_IMAGE** environment variable is set to App Mesh Envoy,
+    ```
+    export ENVOY_IMAGE=840364872350.dkr.ecr.us-east-2.amazonaws.com/aws-appmesh-envoy:v1.12.1.1-prod
```

Comment:
Could you instead link to the [envoy image](https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html) docs and not hardcode any image here?

Reply:
yes